spec/default_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe 'yum-cron::default' do
+  [CENTOS_6_OPTS, CENTOS_7_OPTS].each do |pltfrm|
+    describe "on #{pltfrm[:platform]} #{pltfrm[:version]}" do
+      cached(:chef_run) do
+        ChefSpec::SoloRunner.new(pltfrm).converge(described_recipe)
+      end
+
+      it do
+        expect(chef_run).to install_package('yum-cron')
+      end
+
+      case pltfrm[:version]
+      when '6.7'
```

Comment:
If I do that, then rake gives me this:

Untouched Resources:

  template[/etc/sysconfig/yum-cron]   yum-cron/recipes/default.rb:24
  template[/etc/yum/yum-cron.conf]   yum-cron/recipes/default.rb:24
  template[/etc/yum/yum-cron-hourly.conf]   yum-cron/recipes/default.rb:24